doc/swagger-api-description/api.yaml

```diff
@@ -2,45 +2,44 @@ swagger: '2.0'
 
 info:
   title: ariADDna API
-  description: API for ariADDna project. Use this API you can management all your clouds spaces from single point.
+  description: |
+   #### This document contains the API for ariADDna project.
+   Using this API one can manage all available cloud services (DropBox, GDrive, Yandex.Disk etc.) from single point.
+
   contact:
     name: API Support
     url: http://www.ariaddna.stnetix.com/support
     email: ariaddna.support@stnetix.com
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
-  version: '1.0'
+  version: "1.0"
 
 host: api.ariaddna.stnetix.com
-
+basePath: /v1
 schemes:
   - https
-
+#I guess using 'charset=utf-8' in consumes and produces is a mistake. See MIME types in Swagger spec.
 consumes:
-  - application/json; charset=utf-8
-
+  - application/json
 produces:
-  - application/json; charset=utf-8
-
-basePath: /v1
+  - application/json
 
 paths:
-  /user:
+  /users:
   #Create a new user.
     post:
       description: Creating a new user.
       operationId: addUser
       parameters:
-        - name: user
-          in: body
-          description: Add new user.
+        - in: body
+          name: user
           required: true
           schema:
             $ref: '#/definitions/User'
       responses:
         '201':
-          description: User response.
+          description: A new user was successfully created.
           schema:
             $ref: '#/definitions/User'
         '404':
```

Comment:
I've changed 404 to 409 status code, but I am not sure about such kind of error because POST is non-idempotent request. What are we going to do if someone tries to create user with the already existing login? Shall we use PUT method for creating a new user? 
By the way, which parameter will be the key in the data base table containing User's information? If it is not login then the mentioned above does not matter